app/models/manageiq/providers/vmware/infra_manager/inventory/collector.rb

```diff
@@ -50,7 +50,7 @@ def vim_collector
     _log.info("#{log_header} Monitor updates thread started")
 
     vim = connect
-    property_filter = create_property_filter(vim)
+    property_filter = create_property_filter(vim, ems_inventory_filter_spec(vim))
```

Comment:
This makes create_property_filter more general, allowing creation of property filters for other purposes e.g. events